Traceback (most recent call last):
  File "/home/petri/Python/venv/Jupyter/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/petri/Python/venv/Jupyter/lib/python3.11/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/petri/Python/venv/Jupyter/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/petri/Python/venv/Jupyter/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/petri/Python/venv/Jupyter/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/petri/Python/venv/Jupyter/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# Find the shortest route
%timeit route = ox.shortest_path(G, orig, dest, weight="length")

# Convert to Geopandas Data Frame
gdf=ox.routing.route_to_gdf(G, route)
------------------

----- stdout -----
106 ms Â± 3.05 ms per loop (mean Â± std. dev. of 7 runs, 10 loops each)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[26], line 5[0m
[1;32m      2[0m get_ipython()[38;5;241m.[39mrun_line_magic([38;5;124m'[39m[38;5;124mtimeit[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124mroute = ox.shortest_path(G, orig, dest, weight=[39m[38;5;124m"[39m[38;5;124mlength[39m[38;5;124m"[39m[38;5;124m)[39m[38;5;124m'[39m)
[1;32m      4[0m [38;5;66;03m# Convert to Geopandas Data Frame[39;00m
[0;32m----> 5[0m gdf[38;5;241m=[39mox[38;5;241m.[39mrouting[38;5;241m.[39mroute_to_gdf(G, [43mroute[49m)

[0;31mNameError[0m: name 'route' is not defined

